velox/core/PlanNode.h

```diff
@@ -762,6 +762,58 @@ class TableWriteMergeNode : public PlanNode {
   const RowTypePtr outputType_;
 };
 
+/// For each input row, generates N rows with M additional columns according to
+/// specified 'projections'. 'Projections' is an N x M matrix of expressions.
+/// Each expression is either a column reference or a constant. Both null and
+/// non-null constants are allowed. 'Names' is a list of new column names. It
+/// contains exactly M entries. The semantic of this operator matches Spark.
+class ExpandNode : public PlanNode {
+ public:
+  /// @param id Plan node ID.
+  /// @param projectSets A list of project sets. The output conatins one cloumn
+  /// for each project expr. The project expr may be cloumn reference, null or
+  /// int constant.
+  /// @param names The names and order of the projects in the output.
+  /// @param source Input plan node.
+  ExpandNode(
+      PlanNodeId id,
+      std::vector<std::vector<TypedExprPtr>> projects,
```

Comment:
'projects' name doesn't match 'projections' in class-level comment or 'projectSets' in method-level comment. Let's make sure all these names match and use 'projections'.